Software/Development/SemesterProject/App/systemManager.c

```diff
@@ -42,14 +42,15 @@ void manageSystem(void)
         case STANDBY:
         {
             if(isPowerSwitchEnabled() && userACKHasBeenPressed()) systemState = RUNNING;
+            if(getSystemErrorStatus() == ERROR_HAS_HAPPENED) systemState = ERROR;
```

Comment:
I think this function shouldnt include system as it might be confusing, for example getErroManagerStatus()

Reply:
I will do this, but i removed the typo...

Reply:
( ͡° ͜ʖ ͡°)